src/build/__init__.py

```diff
@@ -40,13 +40,16 @@
 TOMLDecodeError: Type[Exception]
 toml_loads: Callable[[str], MutableMapping[str, Any]]
 
-
 try:
-    from tomli import TOMLDecodeError
-    from tomli import loads as toml_loads
-except ModuleNotFoundError:  # pragma: no cover
-    from toml import TomlDecodeError as TOMLDecodeError  # type: ignore
-    from toml import loads as toml_loads  # type: ignore
+    from tomllib import TOMLDecodeError  # type: ignore
+    from tomllib import loads as toml_loads  # type: ignore
```

Comment:
Rather than this try-except let's use if/else and check based on the python version what we need to do. This is also mypy friendly.